samples/winapi_filefind/main.c

```diff
@@ -10,10 +10,15 @@ int main(void)
 {
     XVideoSetMode(640, 480, 32, REFRESH_DEFAULT);
 
+    // Create a variable for WinAPI error checking
+    DWORD error;
```

Comment:
This can be a highly local variable, I don't see a benefit for putting it at function scope?
If you declare it where it's used it also won't require any additional comment.

If the variable is redeclared, you should use different names to differentiate them; in this particular sample this only seems to happen after `FindNextFile`, so you could probably do `DWORD findFileError = GetLastError()` in that instance.

Reply:
This wasn't done everywhere